torch/distributed/nn/jit/templates/remote_module_template.py

```diff
@@ -17,47 +17,46 @@
 # WARNING: If the module is on a cuda device, any CPU tensor stored in a nested ``arg`` or `kwargs``,
 # will not be implicitly moved to the same cuda device.
 def _remote_forward(
-    module_rref: RRef[module_interface_cls], device: str, {arg_types}){arrow_and_return_type}:
+    module_rref: RRef[module_interface_cls], device: str, is_device_map_set: bool, {arg_types}){arrow_and_return_type}:
```

Comment:
Is `is_device_map_set` argument user-facing?

Reply:
Not user-facing. See L70. We pass a wrapper of (internal args, user-facing args) into `_remote_forward`.